docs/threat-model/threat-model.md

```diff
@@ -68,6 +68,39 @@ In case of Kata, today the devices which we need in the guest are:
 How these devices are utilized varies depending on the VMM utilized. We clarify the default settings provided when integrating Kata
 with the QEMU, Firecracker and Cloud Hypervisor VMMs in the following sections.
```

Comment:
Hi @zvonkok, does Dragonball VMM need to be updated in the threat model ？

Reply:
I need to add Dragonball here as well.

Reply:
Done.